Automattic/wp-calypso
# Site Settings: Remove problematic "blacklist"/"whitelist" language #43395

#### Changes proposed in this Pull Request

* Remove problematic language from site settings including Jetpack Protect, comment author blocking, comment word filters, and a related selector.

#### Testing instructions

* Ensure explicitly allowed IPs in Jetpack Connect still works
* Ensure comment author blocking still works (both that you can block/unblock)
* Ensure adding words to the comment filter still works

Addresses part of #43395
